docs/dotnetcore/introducing-dotnetcore/introducing-dotnetcore.rst

```diff
@@ -0,0 +1,94 @@
+
+This article covers the following topics:
+	- What is .NET Core?
+	- Motivation Behind .NET Core
+	- .NET Core and ASP.NET
+	- .NET Core and NuGet
+	- Additional Reading
+
+What is .NET Core
+^^^^^^^^^^^^^^^^^
+
+.NET Core 5 is a modular runtime and library implementation that includes a subset of the full .NET Framework. Currently it builds and runs on Windows, but support for Linux and Mac is under development. .NET Core consists of a set of libraries, called "CoreFX", and a small, optimized runtime, called "CoreCLR". .NET Core is open-source, so you can follow progress on the project and contribute to it on GitHub:
+
+	- `.NET Core Libraries (CoreFX) <https://github.com/dotnet/corefx>`_
+	- `.NET Core Common Language Runtime (CoreCLR) <https://github.com/dotnet/coreCLR>`_
+
+The CoreCLR runtime is made available via NuGet (Microsoft.CoreCLR); the CoreFX library is also available as a set of individual NuGet packages, factored according to functionality. These packages are named "System.[module]" on `Nuget.org <http://www.nuget.org/>`_.
+
+One of the key benefits of .NET Core is its portability. You can package and deploy the CoreCLR with your application, eliminating your application's dependency on an installed version of the full .NET Framework at the device or server level. You can host multiple applications side-by-side using different versions of the CoreCLR, and upgrade them individually, rather than being forced to upgrade all of them simultaneously.
+
+Another benefit is common access to standard libraries. The CoreFX library includes Collections, Console access, Diagnostics, IO, LINQ, JSON, XML, and regular expression support, just to name a few. Applications targeting a variety of devices and deployment models (such as Windows Desktop, Windows Store, Windows Phone, and ASP.NET) can directly target these libraries, avoiding the need to write wrapper code or conditionally compile based on target platform, as is common today.
+
+Motivation Behind .NET Core
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The fragmentation of .NET and its capabilities across different platforms is the primary motivation for .NET Core. When .NET first shipped in 2002, it was a single framework, but it didn't take long before the .NET Compact Framework shipped, providing a smaller version of .NET designed for mobile devices. Over the years, this exercise was repeated multiple times, so that today there are different flavors of .NET specific to different platforms. Add to this the further fragmentation created by Mono and Xamarin, which target Linux, Mac, and native iOS and Android devices, and the need for a single common Core of .NET that works the same across all of these platforms becomes even more apparent. Since .NET Core is a full open source project, it will no longer be necessary for Mono to maintain its own fork of this technology, and instead Microsoft and the Mono community can collaborate together on .NET Core.
```

Comment:
This is a bit far-reaching. It's not exactly wrong, but it's much more "boil the ocean" than we really have in mind. We're working collaboratively with Mono, but I don't think that Mono is going away any time soon. I believe this is what @migueldeicaza thinks, too.

You are right that the .NET Core _framework_ is intended as a new componentized framework that can be used in a uniform way by all implementations to avoid fragmentation. CoreCLR helps, too, in the sense that it can be used for multiple workloads on multiple chips/OSes, in the way you say, but you don't have to use CoreCLR to get the benefit. It's the framework surface area that is the main point, since it forms the primary contract with user code. So, a CoreCLR and a Mono that both support .NET CoreFX is perfectly fine from a fragmentation standpoint.